vss-accessor/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+use ::prost::Message;
+use reqwest;
+use reqwest::Client;
+use std::error::Error;
+
+use crate::vss::{GetObjectRequest, GetObjectResponse, KeyValue, ListKeyVersionsRequest, ListKeyVersionsResponse, PutObjectRequest, PutObjectResponse};
+use crate::vss_error::VssError;
+
+mod vss_error;
+
+pub mod vss {
+    include!(concat!(env!("OUT_DIR"), "/org.vss.rs"));
```

Comment:
Could you use tabs instead of spaces? That's been our convention for Rust projects.

Reply:
replaced with rustfmt